notebooks/5029_create-history-for-global-workflow.py

```diff
@@ -482,8 +482,10 @@ def extend_with_inversions(indf: pd.DataFrame, cmip_inverse: pd.DataFrame) -> pd
     global_workflow_harmonisation_emissions_l.append(to_keep)
 
 global_workflow_harmonisation_emissions = (
-    pix.concat(global_workflow_harmonisation_emissions_l).sort_index().sort_index(axis="columns")
-).loc[:, :HARMONISATION_YEAR]
+    (pix.concat(global_workflow_harmonisation_emissions_l).sort_index().sort_index(axis="columns"))
+    .loc[:, :HARMONISATION_YEAR]
+    .pix.assign(scenario=HISTORY_SCENARIO_NAME)
```

Comment:
This wasn't there before so there was both "history" and "historical" in the scenario column, which wasn't ideal